cmd/root.go

```diff
@@ -165,6 +177,13 @@ func (mv mapVar) Type() string {
 func init() {
 	cobra.OnInitialize(initConfig)
 
+	RootCmd.SetHelpCommand(&cobra.Command{
+		Use:    "hidden-help-command",
```

Comment:
This still doesn't work.

Reply:
The purpose of this, though, is the *remove* the `help` command in favor of the `--help` flag. As far as I can tell this does accomplish that.